annorxiver_modules/corpora_comparison_helper.py

```diff
@@ -1,81 +1,92 @@
-import csv
-
 from collections import Counter
+import csv
+import string
 
 from lxml import etree
 from mizani.formatters import custom_format, log_format
 import numpy as np
 import pandas as pd
 import plotnine as p9
+import spacy
 import svgutils.transform as sg
+from svgutils.compose import Unit
 from tqdm import tqdm_notebook
 
+from annorxiver_modules.document_helper import dump_article_text
+
 
 def aggregate_word_counts(doc_iterator, disable_progressbar=False):
     """
     This function aggregates the word count tsv files.
     Arguments:
         doc_iterator - a pathlib generator that returns file paths to be parsed
         disable_progressbar - show the progressbar?
-    
+
     Example tsv file:
-     
-    | lemma | pos_tag | dep_tag | count | 
-    | --- | --- | --- | --- | 
+
+    | lemma | pos_tag | dep_tag | count |
+    | --- | --- | --- | --- |
     | genome | NOUN | compound | 23 |
     ...
     """
     global_word_counter = Counter()
-    
+
     for doc in tqdm_notebook(doc_iterator, disable=disable_progressbar):
         with open(doc, "r") as tsvfile:
             reader = csv.DictReader(tsvfile, delimiter="\t")
-            
+
             # create keys with the following types
             # lemma - the token string
             # pos_tag - the part of speech tag
             # dep_tag - the dependency path tag
-            global_word_counter.update({
-                (
-                    row['lemma'], 
-                    row['pos_tag'], 
-                    row['dep_tag']
-                ):int(row['count'])
-                for row in reader
-            })
+            global_word_counter.update(
```

Comment:
woah...cool new pandas function i havesn't seen!

Reply:
That's actually a dictionary function. Not pandas fyi